test/test_reads.py

```diff
@@ -898,49 +992,82 @@ def testORFsEmptySequence(self):
         """
         An AA read of length zero must not have any ORFs.
         """
+        OpenORFs = True
         read = AARead('id', '')
-        orfs = list(read.ORFs())
+        orfs = list(read.ORFs(OpenORFs))
         self.assertEqual(0, len(orfs))
 
     def testORFsEmptySequenceWithStartStop(self):
         """
         An AA read with just a start and stop codon must not have any ORFs.
         """
+        OpenORFs = False
         read = AARead('id', 'M*')
-        orfs = list(read.ORFs())
+        orfs = list(read.ORFs(OpenORFs))
         self.assertEqual(0, len(orfs))
 
+    def testORFsEmptySequenceWithStartStopOpenORFs(self):
+        """
+        An AA read with just a start and stop codon must not have any ORFs.
+        """
+        OpenORFs = True
+        read = AARead('id', 'M*')
+        orfs = list(read.ORFs(OpenORFs))
+        self.assertEqual(1, len(orfs))
+
     def testORFsEmptySequenceWithStart(self):
         """
         An AA read with just a start codon must not have any ORFs.
         """
+        OpenORFs = False
         read = AARead('id', 'M')
-        orfs = list(read.ORFs())
+        orfs = list(read.ORFs(OpenORFs))
```

Comment:
Just pass `False`. Don't make the reader do any more work than needed.

Reply:
done